pkgs/development/python-modules/xmldiff/default.nix

```diff
@@ -19,7 +19,7 @@ buildPythonPackage rec {
     hash = "sha256-GbAws/o30fC1xa2a2pBZiEw78sdRxd2PHrTtSc/j/GA=";
   };
 
```

Comment:
```suggestion
  build-system = [ setuptools ];
```

Reply:
It is (also) needed as a runtime dependency for the executables - should it be added both to `build-system` and `dependencies`? I have verified it needs the latter for sure.

Reply:
Done

Reply:
this is to properly support cross builds ;)